includes/class-experimental-abtest.php

```diff
@@ -101,7 +101,7 @@ protected function fetch_variation( $test_name ) {
 		}
 
 		// Make sure test name is a valid one.
-		if ( ! preg_match( '/^[A-Za-z0-9_]+$/', $test_name ) ) {
+		if ( ! preg_match( '/^\w+$/', $test_name ) ) {
```

Comment:
I'm not sure if this is intended but `\w` can include Unicode characters, depending on PCRE settings. [`[:alnum:]`](https://www.php.net/manual/en/regexp.reference.character-classes.php) will be a better match for original regex.

Reply:
Done in 5cd75bc13d3c4fd7c16a30232a7de65a14855925